staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go

```diff
@@ -192,6 +193,7 @@ func NewCmdDrain(f cmdutil.Factory, ioStreams genericclioptions.IOStreams) *cobr
 	cmd.Flags().DurationVar(&o.drainer.Timeout, "timeout", o.drainer.Timeout, "The length of time to wait before giving up, zero means infinite")
 	cmd.Flags().StringVarP(&o.drainer.Selector, "selector", "l", o.drainer.Selector, "Selector (label query) to filter on")
 	cmd.Flags().StringVarP(&o.drainer.PodSelector, "pod-selector", "", o.drainer.PodSelector, "Label selector to filter pods on the node")
+	cmd.Flags().BoolVar(&o.DisableCordon, "disable-cordon", o.DisableCordon, "Not to cordon node before draining it")
```

Comment:
All the other flags are on `o.drainer`.  Should this one be too?

Reply:
this flag is irrelevant to the draining process, it just tell kubectl not to cordon the node before draining. I think it might not be necessary to add this flag to `o.drainer`.